app.py

```diff
@@ -49,7 +56,47 @@ def asynchronous_push(f_stop):
 asynchronous_push(f_stop)
 
 # Now let's run the app!
-chatbot = pipeline(model="microsoft/DialoGPT-medium")
+prompt = load_prompt(PROMPT_TEMPLATES / "openai_chatgpt.json")
+
+chatbot_1 = ConversationChain(
+    llm=HuggingFaceHub(
+        repo_id="google/flan-t5-xl",
+        model_kwargs={"temperature": 1}
+    ),
+    prompt=prompt,
+    verbose=False,
+    memory=ConversationBufferMemory(ai_prefix="Assistant"),
```

Comment:
This logic is specific to the ChatGPT prompt which uses "Assistant" instead of the "AI" default that `langchain` defines in the conversation. I wonder if we should set "Assistant" as the default for all prompt templates, i.e. "Sparrow -> "Assistant" etc?

Reply:
yes hard to say imo. It might depend if the models were actually trained with prompt templates